<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCustomProfile</class>
 <widget class="QDialog" name="NewCustomProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Custom Profile</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_main">
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPlainTextEdit" name="addPro_plainTextEdit">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="addPro_lineEdit">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QFrame" name="frame">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
            <property name="spacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Profile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="addPro_comboBox">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QFrame" name="frame_buttons">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addPro_OK">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addPro_cancel">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>addPro_lineEdit</tabstop>
  <tabstop>addPro_plainTextEdit</tabstop>
  <tabstop>addPro_OK</tabstop>
  <tabstop>addPro_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="rodinP.qrc"/>
 </resources>
 <connections/>
</ui>
